src/dbseeder/dbseeder.py

```diff
@@ -67,9 +68,15 @@ def seed(self, source, file_location, who):
         programs = self._parse_source_args(source)
 
         for program in programs:
-            seederClass = factory.create(program)
-
-            seeder = seederClass(db, file_location=file_location)
+            seederClass = factory.get(program)
+
+            seeder = seederClass(db=db,
+                                 update=False,
+                                 source=file_location or secrets.sdwis,
```

Comment:
this is going to fall flat when we try to do the fgdb updates. Let's think about how to make the cli and ctor parameters nice and terse.